### ISSUE TITLE
Can i write smart contract for tezos using netezos?

### ISSUE THREAD

**Issue by KamranShahid** [baking-bad/netezos#48]
I was reading https://www.c-sharpcorner.com/article/getting-started-with-smart-contracts-in-c-sharp/ for jump into smart contract creation. 
Is there any similar article targeting tezos?

**Comment by KamranShahid** [comment]
or can i write smart contract for tezos using C#
Any help @Groxan

**Comment by Groxan** [comment]
No, you can't write smart contracts on C#. However, you can try to create your own language with syntax based on C#, and then create a compiler from your language to Michelson.

Netezos is an SDK for interacting with smart contracts and the Tezos blockchain in general.
